terraform-tests/redis_test.go

```diff
@@ -41,6 +39,8 @@ var _ = Describe("Redis", Label("redis-terraform"), Ordered, func() {
 		"maintenance_day":                    nil,
 		"data_tiering_enabled":               false,
 		"multi_az_enabled":                   true,
+		"backup_retention_limit":             12,
+		"final_backup_identifier":            "tortoise",
```

Comment:
I believe we don't have consensus on how to interpret this `defaultVars` variable.

@zucchinidev made a good point in one of our pairing sessions about trying to keep this variable
in sync with the defaults defined in the `aws-redis.yml` manifest.

I believe it is not currently in-sync (probably my fault) but:
Do we want to follow that convention? If so, then these lines should read:
```suggestion
		"backup_retention_limit":             1,
		"final_backup_identifier":            nil,
```

Reply:
A problem with testing default values is that they are not usually unique. So you can't tell the difference between these two lines:
```hcl
  final_snapshot_identifier  = var.final_backup_identifier // passes
  
  final_snapshot_identifier  = var.kms_key_id // also passes
```

Maybe it would also be useful to have a test of the default values in the `aws-redis.yml` file, and we if write that test it should parse the default values from the file rather than have humans keep it in sync.